.prettierignore

```diff
@@ -1,4 +1,5 @@
 # Ignore artifacts:
 build
 coverage
-storybook-static
\ No newline at end of file
+storybook-static
+ISSUE_TEMPLATE
```

Comment:
The ISSUE_TEMPLATE should be uploaded to GitHub, otherwise it won't work :).

Reply:
THX! I solved the problem and have prettify the issue template :-)